patches/nginx-1.15.8-ssl_pending_session.patch

```diff
@@ -1,16 +1,36 @@
 --- nginx-1.15.8/src/event/ngx_event_openssl.c	2016-07-17 19:20:30.411137606 -0700
 +++ nginx-1.15.8-patched/src/event/ngx_event_openssl.c	2016-07-19 16:53:35.539768477 -0700
-@@ -1307,7 +1307,12 @@ ngx_ssl_handshake(ngx_connection_t *c)
+@@ -1307,7 +1307,17 @@ ngx_ssl_handshake(ngx_connection_t *c)
```

Comment:
minor: On NGINX's `release-1.15.8` tag with the `nginx-1.15.8-ssl_cert_cb_yield` patch already applied, I get different line numbers for this chunk once I apply this patch:
```
@@ -1446,7 +1446,17 @@ ngx_ssl_handshake(ngx_connection_t *c)
```

This doesn't seem to prevent the patch from applying, but worth mentioning.